main.nf

```diff
@@ -2061,7 +2061,8 @@ process SPADES_VG {
     samtools faidx $fasta
     for chrom in `cat ${fasta}.fai | cut -f1`
     do
-        vg deconstruct -p \$chrom ${sample}.xg --threads $task.cpus \\
+        vg snarls ${sample}.xg > ${sample}.snarls
```

Comment:
This can be placed outside of the loop right? Just needs to be generated once as opposed to per chromosome.

Reply:
Yes, you're right!  Good catch!